maximus-two-wilfinitlike.gmail.com/decoration.js

```diff
@@ -195,7 +247,49 @@ function onChangeNWorkspaces() {
 /*
  * Subextension hooks
  */
-function init() {}
+function init() {
+  _loadSettings();
+
+  this._settingsListenerId = this._settingsObject.connect(
+        'changed::ignorable-apps',
+        Lang.bind(this, this._settingsChanged)
+      );
+}
+
+function _settingsChanged() {
+	this.ignorableApps = JSON.parse(this._settingsObject.get_string("ignorable-apps"));
+
+
+	let apps = global.display.get_tab_list(Meta.TabList.NORMAL_ALL, null);
+
+
+	for (let i = 0; i < apps.length; i++) {
+		_doHideTitlebar(apps[i], !this._isIgnorebleWindow(apps[i].get_title()));
+	}
+}
+
+function _loadSettings() {
```

Comment:
Imported from prefs.js or another place to avoid the duplication it generates with [this](https://github.com/wilfm/GnomeExtensionMaximusTwo/pull/27/files#diff-cd312dcc7f2b5902c0febbe5800c6b57R183). May be there should be a middle ground between decoration.js <-> prefs.js?